### PR TITLE
Am I Under Attack - User doc

### PR DESCRIPTION
**Pull request by TheCrowdsecMatt**

### CODE REVIEW

File: crowdsec-docs/static/img/console/am-i-under-attack/am-i-under-attack-settings.png
Comment (LaurenceJJones) [inline crowdsec-docs/static/img/console/am-i-under-attack/am-i-under-attack-settings.png:1]: Is the picture meant to show `link to update` or should I not take it into account?

File: crowdsec-docs/unversioned/console/am-i-under-attack.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Am I Under Attack
+
+## Introduction
+
+The feature Am I Under Attack is the pinnacle of what CrowdSec offers in terms of real-time infrastructure monitoring.
```
Comment (LaurenceJJones) [inline crowdsec-docs/unversioned/console/am-i-under-attack.md:5]: I dont generally like this wording "pinnacle", has mkt approved this? if not can I refactor?

File: crowdsec-docs/unversioned/console/am-i-under-attack.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+If the situation occurs, multiple communication channels will be used to reach out owner and administrator members of the organization.
+
+*   An email will be sent (max 1 / day)
+*   In-app notifications will be sent to the Console accounts (max 1/hour)
+*   A warning banner will be displayed on the Console accounts.
+
+  
+
+These inputs will contain a link to the alert page of the Console to get a precise overview giving the possibility to assess the situation and quickly react accordingly.
```
Comment (LaurenceJJones) [inline crowdsec-docs/unversioned/console/am-i-under-attack.md:19]: Can we specify what the link contains EG predefined filter of timeline etc etc?

File: crowdsec-docs/unversioned/console/am-i-under-attack.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+  
+
+## Pre-requisites 
+
+*   The feature is only available for __Enterprise plan users__.
```
Comment (LaurenceJJones) [inline crowdsec-docs/unversioned/console/am-i-under-attack.md:25]: Could we link to pricing page like EG:

```
This is an [enterprise plan](https://www.crowdsec.net/pricing) feature
```

File: crowdsec-docs/static/img/console/am-i-under-attack/am-i-under-attack-settings.png
Comment (TheCrowdsecMatt) [reply crowdsec-docs/static/img/console/am-i-under-attack/am-i-under-attack-settings.png:1]: You're right, I have to replace it!

File: crowdsec-docs/unversioned/console/am-i-under-attack.md (hunk 1 above)
Comment (TheCrowdsecMatt) [reply crowdsec-docs/unversioned/console/am-i-under-attack.md:5]: No that's from me, you can rephrase it if you think it's not really suitable

File: crowdsec-docs/unversioned/console/am-i-under-attack.md (hunk 3 above)
Comment (TheCrowdsecMatt) [reply crowdsec-docs/unversioned/console/am-i-under-attack.md:25]: It's not available yet, as the feature is not publicly advertised yet.
But sure we'll have to change it for the official release

**Comment by rr404** [comment]
I made a new PR for the latest release of the AIUA : https://github.com/crowdsecurity/crowdsec-docs/pull/635
maybe this renders this PR obsolete
@TheCrowdsecMatt

**Comment by LaurenceJJones** [comment]
Guess this closes as #635 took preference?

**Pull request closed without merging**